app/forms/support_interface/block_submission_form.rb

```diff
@@ -11,11 +11,9 @@ def save(fraud_match_id)
 
       return false unless valid?
 
-      FraudMatch.find(fraud_match_id).candidates.each do |candidate|
-        return unless candidate.update(submission_blocked: true)
+      FraudMatch.find(fraud_match_id).candidates.all? do |candidate|
```

Comment:
... and is it a bit weird to side-effect in an `all?` block?

Reply:
Yes, I think the explicit `return unless` in an `each` block is easier to follow (to me) so will change...